couscous.yml

```diff
@@ -0,0 +1,38 @@
+template:
+    url: https://github.com/CouscousPHP/Template-Dark
+
+exclude:
+    - library
+    - vendor
+    - bin
+    - tests
+
+baseUrl: http://respect.github.io/Validation
+
+github:
+    user: Respect
+    repo: Validation
+
+title: Respect\Validation
+subTitle: The most awesome validation engine ever created for PHP
+
+menu:
+    sections:
+        main:
+            name: Table of contents
+            items:
+                home:
+                    text: Home
+                    relativeUrl: /
+                contributing:
+                    text: Contributing
+                    relativeUrl: CONTRIBUTING.html
+                documentation:
```

Comment:
Yes, on `sections` you can have more than one menu.
I can do it if you like, the only problem is because the links will broken on GitHub. Do you think it worth?